src/app/components/search/search.component.ts

```diff
@@ -21,7 +21,7 @@ export class SearchComponent implements OnInit {
   ngOnInit(): void {
     this.searchString$.pipe(
       tap(this.checkRedirection.bind(this)),
-      debounceTime(500),
+      debounceTime(1000),
```

Comment:
Is it reasonable to increase it to 1500? What do you think?

Reply:
I think 1000 is enough